packages/contracts-bedrock/src/dispute/FaultDisputeGame.sol

```diff
@@ -507,10 +510,20 @@ contract FaultDisputeGame is IFaultDisputeGame, Clone, ISemver {
             return;
         }
 
+        // Fetch the resolution checkpoint from storage.
+        ResolutionCheckpoint memory checkpoint = resolutionCheckpoints[_claimIndex];
+
+        // If the checkpoint does not currently exist, initialize the current left most position as max u128.
+        if (checkpoint.claimIndex == 0) {
```

Comment:
This doesn't overwrite the checkpoint progress of an absolute root claim resolution?